exporters/otlp/include/opentelemetry/exporters/otlp/recordable.h

```diff
@@ -19,6 +19,9 @@ class Recordable final : public sdk::trace::Recordable
 public:
   const proto::trace::v1::Span &span() const noexcept { return span_; }
 
+  /** Dynamically converts the resource of this span into a proto. */
+  proto::resource::v1::Resource resource() const noexcept;
```

Comment:
Probably call it `proto_resource` to differentiate it with SDK resource?

Reply:
Sure, changed it.